src/ui/FileViewer/DeleteButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FileViewerComponentProps, ViewerTypes } from './types';
+import Icon, { IconColors, IconTypes } from '../Icon';
+
+export function DeleteButton(props: FileViewerComponentProps): React.ReactElement {
```

Comment:
If we will export this UI component, it will be good to accept only necessary props
However, it's for internal for now, so we can fix it later